lib/config/helper_test.go

```diff
@@ -364,29 +364,19 @@ func TestValidateS3Endpoint(t *testing.T) {
 	}{
 		{
 			name:          "Valid Endpoint",
-			endpoint:      "s3-external-1.amazonaws.com",
-			expectedError: false,
+			endpoint:      "",
```

Comment:
why did you remove the value ?
Just expect error should be changed to false ?

Reply:
its the unit test check that if endpoint is empty then it will throw an error